### PR TITLE
change PAU to RES authorization for Ogone

### PR DESCRIPTION
**Pull request by adamgrad**

### CODE REVIEW

**Comment by adamgrad** [comment]
@Azdaroth changeset is confusing but I had to merge: https://github.com/BookingSync/active_merchant/pull/6 here in order to not break specs for Stripe on our side. So we need to merge https://github.com/BookingSync/active_merchant/pull/6 first and then this one.

File: test/remote/gateways/remote_stripe_payment_intents_test.rb [hunk 1]
```diff
@@ -0,0 +1,340 @@
+require 'test_helper'
+
+class RemoteStripeIntentsTest < Test::Unit::TestCase
```
Comment (Azdaroth) [inline test/remote/gateways/remote_stripe_payment_intents_test.rb]: it's the same as in the other PR, right?

**Review by Azdaroth — APPROVED** [review]
💪 🐉 Opa!

Assuming that Stripe changes are the same as from the other PR

File: test/remote/gateways/remote_stripe_payment_intents_test.rb (hunk 1 above)
Comment (adamgrad) [reply test/remote/gateways/remote_stripe_payment_intents_test.rb]: @Azdaroth yeah, that's why I linked the PR. Here I changed only Ogone and it's specs.

**Pull request merged**